src/olympia/addons/serializers.py

```diff
@@ -13,8 +14,8 @@
 
 class FileSerializer(serializers.ModelSerializer):
     url = serializers.SerializerMethodField()
-    platform = serializers.ReadOnlyField(source='get_platform_display')
-    status = serializers.ReadOnlyField(source='get_status_display')
+    platform = ReverseChoiceField(choices=amo.PLATFORM_CHOICES_API.items())
+    status = ReverseChoiceField(choices=amo.STATUS_CHOICES_API.items())
```

Comment:
interesting idea, I like it. Although, if we could add reversed items for `PLATFORM_CHOICES_API` and  `STATUS_CHOICES_API` we could safe a few microseconds for every serialization. I honestly don't care though and like it :)